norse/torch/functional/test/test_lif.py

```diff
@@ -2,27 +2,28 @@
 import pytest
 
 import norse
-import norse.torch.functional.lif as lif_module
 from norse.torch.functional.lif import (
     LIFState,
     LIFFeedForwardState,
     LIFParameters,
     LIFParametersJIT,
     lif_step,
+    # lif_step_integral,
```

Comment:
comment?

Reply:
Good catch! I added the module, including some tests.